nbgrader/apps/assignapp.py

```diff
@@ -12,103 +13,152 @@
     LockCells,
     ComputeChecksums,
     SaveCells,
-    CheckCellMetadata
+    CheckCellMetadata,
+    ClearOutput,
 )
 
 aliases = {}
 aliases.update(nbconvert_aliases)
 aliases.update({
-    'assignment': 'AssignmentExporter.assignment_id',
-    'db': 'AssignmentExporter.db_url'
 })
 
 flags = {}
 flags.update(nbconvert_flags)
 flags.update({
-    'save-cells': (
-        {'AssignApp': {'save_cells': True}},
-        "Save information about grade cells into the database."
+    'no-db': (
+        {'SaveCells': {'enabled': False}},
+        "Do not save information about grade cells into the database."
+    ),
+    'create': (
+        {'AssignApp': {'create_assignment': True}},
+        "Create the assignment at runtime if it does not exist."
```

Comment:
How about "Create an entry for the assignment in the database, if one does not already exist"?